src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -310,6 +311,14 @@ class SourcesTree extends Component<Props, State> {
       highlightItems
     } = this.state;
 
+    const onExpand = (item, expandedState) => {
```

Comment:
Both the functions are the same so how about `toggleExpanded`?

Reply:
haha - it's small enough where this is fine too. Good either way